### DISCUSSION TITLE
Create new adonis project doesn't generate User and Token models/migrations

### DISCUSSION THREAD

**Discussion by ryanwalker** [adonisjs/core#3321] [Help] [answered]
I'm trying to setup a brand new adonis project and use the api tokens functionality. Reading here: https://docs.adonisjs.com/guides/auth/introduction#migrations 
This says 
```
The setup process also creates the migration files for the users table and optionally for the tokens table (if using the API tokens guard with SQL storage).
```
When I setup a brand new project, there are nor User or Token model/migrations. I can't for the life of me find in the docs on how to generate these files. 
Anyone know how to do this?
Thanks in advance for the help.

**Comment by thetutlage** [comment, marked as answer]
- Scroll to the top of the page ( same link as you shared )
- Click the `Configure` tab to view the command to run

<img width="804" alt="CleanShot 2021-11-10 at 07 21 45@2x" src="https://user-images.githubusercontent.com/1706381/141034986-9e1a86cd-edf6-4261-8bc2-bc212546cdbf.png">

**Reply by ryanwalker** [reply]
oh it's right there. I went through that page a few times and never realized that was there. thanks so much!
